packages/vscode-extension/src/project/tools.ts

```diff
@@ -40,9 +40,9 @@ export interface ToolPlugin extends Disposable {
   persist: boolean;
   pluginAvailable: boolean;
   pluginUnavailableTooltip?: string;
-  activate(): void;
+  enable(): void;
```

Comment:
I think we should have both `activate()` and `enable()` methods for symmetry -- it seems weird to me that you call `deactivate()` to turn it off, but `enable()` to turn it back on.